### PR TITLE
Updated to bevy 0.10

### PR DESCRIPTION
**Pull request by zwazel**
Hi, I quickly updated it so that it works in Bevy 0.10.
This is my first time making any contribution, so please correct me if I did something wrong.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -12,11 +12,11 @@ keywords = ["bevy", "gamedev", "steam"]
 default = []
 
 [dependencies]
-bevy_log = "0.9"
```
Comment (johanhelsing) [inline Cargo.toml:15]: Any reason to prefer full versions with patch?

**Review by johanhelsing — APPROVED** [review]
Not a maintainer, but looks good to me. Running fine in my project.

File: src/lib.rs [hunk 2]
```diff
@@ -221,10 +221,9 @@ fn add_event<T: Callback + Send + Sync + 'static>(
...
             pending,
         })
-        .add_system_to_stage(
-            CoreStage::PreUpdate,
+        .add_system(
             flush_events::<T>
-                .label(SteamworksSystem::FlushEvents)
-                .after(SteamworksSystem::RunCallbacks),
+                .in_set(SteamworksSystem::FlushEvents)
+                .after(SteamworksSystem::RunCallbacks)
```
Comment (johanhelsing) [inline src/lib.rs:227]:
```suggestion
                .after(SteamworksSystem::RunCallbacks),
```
Linting wants this

File: src/lib.rs [hunk 3]
```diff
@@ -152,9 +152,9 @@ impl Plugin for SteamworksPlugin {
             Ok((client, single)) => {
                 app.insert_resource(Client(client.clone()))
                     .insert_non_send_resource(single)
-                    .add_system_to_stage(
-                        CoreStage::First,
-                        run_steam_callbacks.label(SteamworksSystem::RunCallbacks),
+                    .add_system(run_steam_callbacks
+                        .in_base_set(CoreSet::First)
+                        .in_set(SteamworksSystem::RunCallbacks)
```
Comment (johanhelsing) [inline src/lib.rs:157]:
```suggestion
                        .in_set(SteamworksSystem::RunCallbacks),
```

**Review by james7132 — APPROVED** [review]

File: src/lib.rs [hunk 4]
```diff
@@ -152,9 +152,9 @@ impl Plugin for SteamworksPlugin {
             Ok((client, single)) => {
                 app.insert_resource(Client(client.clone()))
                     .insert_non_send_resource(single)
-                    .add_system_to_stage(
-                        CoreStage::First,
-                        run_steam_callbacks.label(SteamworksSystem::RunCallbacks),
+                    .add_system(run_steam_callbacks
+                        .in_base_set(CoreSet::First)
+                        .in_set(SteamworksSystem::RunCallbacks),
```
Comment (johanhelsing) [inline src/lib.rs:157]:
```suggestion
                    .add_system(
                        run_steam_callbacks
                            .in_base_set(CoreSet::First)
                            .in_set(SteamworksSystem::RunCallbacks),
```
Aaah, I botched it. Try again :P ?

**Pull request merged**